docs/source/peer-chaincode-devmode.md

```diff
@@ -33,7 +33,13 @@ Throughout this tutorial, all commands are performed from the `fabric/` folder.
     ```
     export FABRIC_CFG_PATH=$(pwd)/sampleconfig
     ```
-5. Generate the genesis block for the ordering service. Run the following command to generate the genesis block and store it in `$(pwd)/sampleconfig/genesisblock` so that it can be used by the orderer in the next step when the orderer is started.
+5. Create the `hyperledger` subdirectory in the `/var` directory. This is a location the orderer uses to store blocks as defined in the      `orderer.yaml`. To create the `hyperledger` subdirectory, execute these commands, replacing the question marks with your username:
```

Comment:
Both orderer (orderer.yaml) and peer (core.yaml) are configured to store ledger in /var/hyperledger by default.

Reply:
they use it but they don't create it.

Reply:
You'd get the same issue if peer was started without /var/hyperledger existing, so I think if we are educating people about the orderer need, we should also mention the peer need.

Reply:
Following the logic of the guide, the orderer is launched first so by follow the instructions (with /var/hyperledger creation) there is no need to repeat the same for the peer.

Reply:
I understand, I'm not suggesting to repeat it, I'm just suggesting to mention "orderer (orderer.yaml) and peer (core.yaml)" in your explanation sentence to better educate people.

Reply:
Fair enough

Reply:
@denyeart I've addressed your request